renderers/main.py

```diff
@@ -0,0 +1,133 @@
+from data.final import Final
```

Comment:
Some of these imports definitely aren't being used (mlbgame at least), could you remove?

Reply:
I removed a bunch of imports. The only one I wasn't 100% sure about was math but I wasn't able to find anywhere where we were using it so hopefully nothing breaks down.